### DISCUSSION TITLE
Github releases/tags missing

### DISCUSSION THREAD

**Discussion by jacobsvante** [DioxusLabs/dioxus#1012] [Ideas]
I noticed that the Dioxus 0.3.x versions do not exist as a github releases or tags, last one available is 0.2.4. Would be great to have when inspecting code changes between versions. Is this something you would consider adding?

**Comment by ealmloff** [comment]
I agree, they are also useful to view old examples that work with the published version as we start to merge breaking changes for `0.4`. I just added a release tag for `v0.3.2`. Hope that helps!

**Comment by jacobsvante** [comment]
Thanks @Demonthos!

Have you thought about using something like [Google's release-please](https://github.com/googleapis/release-please) to automate the release/tag creation process? It works like a charm in tandem with cargo publish. And if you're like me and don't enjoy writing up changelogs manually you'll be in for a treat!
